resources/views/annotations/show/tabs/settings.blade.php

```diff
@@ -60,6 +60,10 @@
             <power-toggle :active="measureTooltip" title-off="Show measure tooltip" title-on="Hide measure tooltip" v-on:on="showMeasureTooltip" v-on:off="hideMeasureTooltip">Measure Tooltip</power-toggle>
         </div>
 
+        <div class="sidebar-tab__section">
+            <power-toggle :active="restrictToBounds" title-off="Enable annotation boundary restriction" title-on="Disable annotation boundary restriction" v-on:on="enableRestrictToBounds" v-on:off="disableRestrictToBounds">Restrict to Boundaries</power-toggle>
+        </div>
+
```

Comment:
Make the "on" title: "Enable restricting new annotation points to the image boundaries" and the "off" title: "Disable...".